installers/ad-synchrona-14-leds.sh

```diff
@@ -0,0 +1,83 @@
+#!bin/bash
+
+HMC7044_STATUS_PATH="/sys/kernel/debug/iio/iio:device0/status"
+AD9545_STATUS_PATH="/sys/kernel/debug/clk/PLL1/PLL1"
+
+RED_LED1=12
+GREEN_LED1=16
+
+STATUS_RED=0
+STATUS_RED_BLINKING=1
+STATUS_GREEN_BLINKING=2
+STATUS_RED_GREEN_BLINKING=3
+STATUS_GREEN=4
+
+LED_STATUS=$STATUS_RED
+AD9545_LOCKED=0
+HMC7044_LOCKED=0
+AD9545_FREERUN=0
+
+while true
+do
+	ad9545_pll_status=$(cat $AD9545_STATUS_PATH)
+	hmc7044_status=$(cat $HMC7044_STATUS_PATH)
```

Comment:
nit: just realized this now.... You could  use libiio tools to read the status and so you would not need to make assumptions on the device path. The whole thing already depends on libiio anyways so I think it's fine to assume it is installed. No need to do the change though